### PR TITLE
adding the vertical line at the last day of measurement

### LINKED ISSUE #76: Add vertical line at last measurement date to show when the data stops
This would make it clearer when measurement data stops. 

How to do this: https://plot.ly/python/shapes/#lines-positioned-relative-to-the-plot--to-the-axes (xref=x, yref=paper).

### PR DESCRIPTION
**Pull request by mrastgoo**
closes #76

### CODE REVIEW

File: make_figures.py [hunk 1]
```diff
@@ -179,6 +181,24 @@ def make_timeplot(df_measure, df_prediction):
...
     )
+    # vartical line to seprate the last day of measurements from prediction
+    fig.add_shape(
+        # Line Vertical
+        dict(
+            type='line',
+            xref='x',
+            yref='paper',
+            x0=last_day,
+            y0=0,
```
Comment (emmanuelle) [inline make_figures.py]: maybe aesthetically it would be lighter to start a bit above 0, like `y0=0.1` and `y1=0.9`.

**Comment by emmanuelle** [comment]
Thank you very much! I left a very minor comment, but we can also merge as it is if you don't have the time to answer / address this.

**Comment by mrastgoo** [comment]
Hi, I set the limit to 0.05 - 0.95 instead of 0.1 - 0.9 but if you think i should reduce it more let me know

**Comment by GaelVaroquaux** [comment]
This looks good to me. Merging.

**Pull request merged**